src/AppInstallerCommonCore/Public/winget/Experiment.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include <string>
+#include <type_traits>
+#include "Experiment/Experiment.h"
+#include "AppInstallerStrings.h"
+
+namespace AppInstaller::Settings
+{
+    enum ExperimentToggleSource
+    {
+        Default = 0,
+        Policy,
+        UserSettingIndividualControl,
+        UserSettingGlobalControl,
+    };
+
+    struct ExperimentState
+    {
+        ExperimentState() = default;
+        ExperimentState(bool isEnabled, ExperimentToggleSource toggleSource) : m_isEnabled(isEnabled), m_toggleSource(toggleSource) {}
+
+        /// <summary>

```

Comment:
We don't use this doc string format in C++. I think we just add the comment above.